tasks.js

```diff
@@ -12,41 +13,72 @@ module.exports = (resources) => {
     let output = {}
 
     try {
-      const data = await setup(await validate( await load(input)))
-      // business logic goes here...
-      // output = ...
-    } catch (e) { 
-      // if (e.message === ...
-      //   throw new Error('...
-      // }
-      // Default case
-      _debug(e.stack)
-      throw (e)
+      const data = await setup(await validate(await load(input)))
+      output = data.files + data.errored_files
```

Comment:
What is the goal with this? If you add 2 different arrays you will concat them as strings, and then the output will not be as expected, it should return an array.

Reply:
You could use a spread operator if you want
```suggestion
      output = [...data.files , ...data.errored_files}
```

Reply:
Also, you can use the `output = data.files.concat(data.errored_files)`